### DISCUSSION TITLE
Said nightly build link?

### DISCUSSION THREAD

**Discussion by bpowder1966** [BimmerGestalt/AAIdrive#390] [General]
Hello, I used to have a link to the nightly build but it no longer works. Is there a new link?

Thanks!!

**Comment by hufman** [comment]
Yes indeed! As part of moving and renaming the project, the nightly build link has changed too. The Readme has been updated with the new links: [sentry](https://bimmergestalt.s3.amazonaws.com/aaidrive/builds/androidautoidrive-latest-main-nomap-sentry-release.apk) and [nonalytics](https://bimmergestalt.s3.amazonaws.com/aaidrive/builds/androidautoidrive-latest-main-nomap-nonalytics-release.apk).
